go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Microsoft/go-winio v0.4.16 // indirect
 	github.com/antonmedv/expr v1.8.9
 	github.com/appleboy/gin-jwt/v2 v2.6.4
+	github.com/aws/aws-sdk-go v1.34.28
```

Comment:
It seems that a more recent version exists. Is this intended ?

Reply:
even more, we have the topic of migrating to aws sdk v2, but it's go >= 1.15